docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.8'
+
+services:
+  frontend:
+    build:
+      context: .
+      dockerfile: frontend/Dockerfile
+    container_name: arisutalk-frontend
+    restart: unless-stopped
+    ports:
+      - "5173:5173"
+    volumes:
```

Comment:
We should remove `volumes` decrements since docker image contains all of its required files.